packages/@adobe/spectrum-css-temp/components/statuslight/skin.css

```diff
@@ -82,10 +82,10 @@ governing permissions and limitations under the License.
 
 @media (forced-colors: active) {
   .spectrum-StatusLight {
-    forced-color-adjust: none;
     --spectrum-statuslight-text-color-disabled: GrayText;
     --spectrum-statuslight-dot-color-disabled: GrayText;
     &:before {
+      forced-color-adjust: none;
```

Comment:
agreed, this should be here if we're trying to keep the original status light color, however, should it have the border ButtonText? I found it really hard to see the status light with that border. The indicator color was completely washed out by the bright border for me. Is there any case where not showing the status light color and just letting it be hollow or solid ButtonText would make more sense?

Reply:
@jnurthen any thoughts on this?